docs/repo-docs/reference/ls.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: ls
+description: API reference for the `turbo ls` command
+---
+
+import { ExperimentalBadge } from '#/components/experimental-badge';
+
+List packages in your monorepo.
+
+```bash title="Terminal"
+turbo ls [package(s)] [flags]
+```
+
+When scoped to the entire repository, output includes package manager, package count, and all package names and directories.
+
+```bash title="Terminal"
+# List all packages in the repository
+turbo ls
+```
+
+When scoped to one or more packages, output includes package name, directory, internal dependencies, and all tasks.
+
+```bash title="Terminal"
+# List only two packages
+turbo ls [package(s)]
```

Comment:
I think it makes sense to specify the packages here instead of leaving the arg placeholder if the comment references listing 2 packages.
```suggestion
# List only two packages
turbo ls web docs
```